### DISCUSSION TITLE
What can be done in Tasmota for Sonoff devices with LCD screens?

### DISCUSSION THREAD

**Discussion by wayner9** [arendst/Tasmota#19947] [Q&A]
I have a Sonoff THR320D which has a LCD screen which displays the temperature, or perhaps humidity if you have the right probe.

With programming in Tasmota could you display other stuff on the screen if you wanted?  Say the relay status?  Or IP address of the device?  Or anything else?  Or can you just send a number to the screen display driver?

**Comment by sfromis** [comment]
While there is no generic support, the specialized driver does provide indirect support for controlling what numbers to display. Yes, numbers.

You can create a Berry driver to put values into the telemetry data, among which you then can select using the `DspLine` command. Also, there are commands `GlobalTemp`, `GlobalHum` and `GlobalPress` to set "global" values, and with `SetOption2` you can have those in the telemetry data and picked up by the display updating driver.
